rust/otap-dataflow/configs/fake-debug-noop-telemetry.yaml

```diff
@@ -41,5 +41,20 @@ service:
         - periodic:
             exporter:
               console:
+        - periodic:
```

Comment:
I think just one is enough for this config. No need to show all possible options.